RadixWallet/Features/SettingsFeature/AccountSecurity/ManualAccountRecoveryScan/ManualAccountRecoverySeedPhrase+Reducer.swift

```diff
@@ -81,7 +81,7 @@ public struct ManualAccountRecoverySeedPhrase: Sendable, FeatureReducer {
 			let title = state.isOlympia ? "Enter Legacy Seed Phrase" : "Enter Seed Phrase" // FIXME: Strings
 
 			let persistStrategy = ImportMnemonic.State.PersistStrategy(
-				factorSourceKindOfMnemonic: .onDevice(state.isOlympia ? .olympia : .babylon),
+				factorSourceKindOfMnemonic: .onDevice(state.isOlympia ? .olympia : .babylon(isMain: false)),
```

Comment:
Here is the bug fix: the `static func babylon` function always marked all BDFS as `Main`, which resulted in Multiple `Main`! which is not allowed and resulted in `assertionFailure` (we should keep on using `assertionFailure` for weird state, it caught this bug!)